docs/hybrid-cloud.rst

```diff
@@ -125,6 +125,26 @@ Set Up Confluent CLI and variables
       && echo "Schema Registry Cluster ID: $CC_SR_CLUSTER_ID" \
       && echo "Schema Registry Endpoint: $CC_SR_ENDPOINT"
 
+#. Grant the ``ResourceOwner`` role to the cp-demo service account for all Schema subjects.
+
+   .. code:: shell
```

Comment:
We should avoid using Bash highlighting for CLI commands, since it has the tendency to highlight random bash keywords within the commands, like "input".
```suggestion
   .. code:: text
```